### PR TITLE
Store some metadata about the profile in a reserved string table slot

### LINKED ISSUE #15: Store some metadata about the profile in a reserved string table slot.
It might be useful to store data about a collected profile with the profile itself (like process-id, start-time). It occurred to me that we could just pack this data into a JSON string and store that under a pre-reserved `StringId` in the string table. Then we don't need to implement this in the more cumbersome binary format. 

cc @wesleywiser

### PR DESCRIPTION
**Pull request by wesleywiser**
Fixes #15

### CODE REVIEW

**Comment by wesleywiser** [comment]
Fixed and added some additional docs.

File: measureme/src/profiler.rs [hunk 1]
```diff
@@ -42,11 +42,26 @@ impl<S: SerializationSink> Profiler<S> {
...
+        };
+
+        let mut args = String::new();
+        for arg in std::env::args() {
+            args.push_str(&arg.escape_default().to_string());
+            args.push(' ');
+        }
+
+        profiler.string_table.alloc_metadata(&*format!(
+            "{{ \"start_time\": {}, \"process_id\": {}, \"cmd\": \"{}\" }}",
```
Comment (michaelwoerister) [inline measureme/src/profiler.rs]: This might be a perfect place for using raw string literals.
https://rahul-thakoor.github.io/rust-raw-string-literals/

Comment (wesleywiser) [reply measureme/src/profiler.rs]: That's a great idea.

**Comment by amanjeev** [comment]
Ah! I had forgotten. :( Thank you @wesleywiser

**Pull request merged**